### PR TITLE
050 docs updates

### PR DESCRIPTION
**Pull request by TheTrainingBoss**
# 050 docs updates

## Details on the issue fix or feature implementation

Documentation Updates

## Confirm the following

- [x]  I started this PR by branching from the head of the default branch
- [x]  I have targeted the PR to merge into the default branch
- [ ]  I have included unit tests for the issue/feature
- [ ]  I have included inline docs for my changes, where applicable
- [ ]  I have successfully run a local build
- [ ]  I have provided the required update scripts, where applicable
- [x]  I have updated relevant docs, where applicable

> [!NOTE]
> Instead of adding `X`'s inside the checkboxes you wish to check above, first submit the PR, then check the boxes in the rendered description.

### CODE REVIEW

File: docs/deployment/changes-050.md
Comment (saimachi) [inline docs/deployment/changes-050.md:1]: Correct the casing of `Admin_GROUP_OBJECT_ID` on line 105. Explicitly specify that this refers to Entra groups.

File: docs/toc.yml
Comment (saimachi) [inline docs/toc.yml:1]: I would remove the Vectorization Entra ID setup document and Table of Contents entry, given that Vectorization won't be using Entra in the future.

**Review by saimachi — CHANGES_REQUESTED** [review]

File: docs/deployment/authentication-authorization/management-authentication-setup-entra.md
[the remark this answers is not in the document]
Comment (TheTrainingBoss) [reply docs/deployment/authentication-authorization/management-authentication-setup-entra.md:1]: It needs to be done

File: docs/deployment/authentication-authorization/authorization-setup-entra.md
[the remark this answers is not in the document]
Comment (TheTrainingBoss) [reply docs/deployment/authentication-authorization/authorization-setup-entra.md:1]: what is "dedent"?  do you mean "delete"

**Review by ciprianjichici — APPROVED** [review]

**Review by ciprianjichici — APPROVED** [review]

**Review by joelhulen — APPROVED** [review]

**Pull request merged**